index.html

```diff
@@ -321,6 +321,97 @@ <h2>Securing JSON-LD VCs with COSE</h2>
+
+  <section>
+    <h2>Registered Claim Names</h2>
+    <p>
+When present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>, or 
+the <a data-cite="RFC7519#section-4.1.1">Protected Claimset</a> members present in
+<a href="https://www.iana.org/assignments/jwt/jwt.xhtml">IANA Assignments for JSON Web Token (JWT)</a> and 
+<a href="https://www.iana.org/assignments/jose/jose.xhtml">IANA Assignments for JSON Object Signing and Encryption (JOSE)</a>
+are to be interpreted according to the associcated specifications referenced by IANA.
+    </p>
+    <p>
+<a href="#registered-claim-names">Registered claims</a> that are present in either the <a data-cite="RFC7515#section-4.1">Protected Header</a>, 
+or the <a data-cite="RFC7519#section-4.1.1">Claimset</a> can be used to help
+<a data-cite="VC-DATA-MODEL#dfn-verifier">verifiers</a> discover verification keys.
+    </p>
+    <section>
+      <h2>kid</h2>
+      <p>
+If <code>kid</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>,
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
+to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
+<a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
+      </p>
+    </section>
+    <section>
+      <h2>iss</h2>
+      <p>
+If <code>iss</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>,
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
+to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
+<a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
+      </p>
+      <p>
+      <p>
+If <code>iss</code> is present in the <a data-cite="RFC7519#section-4.1.1">JWT Claims </a>,
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
+to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
+<a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
+      </p>
+If <code>kid</code> is also present, it is expected to be useful to distinguish the specific key used.
+      </p>
+    </section>
+
+    <section>
+      <h2>cnf</h2>
+      <p>
+If <code>cnf</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>,
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
+to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
+<a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
+      </p>
+      <p>
+      <p>
+If <code>cnf</code> is present in the <a data-cite="RFC7519#section-4.1.1">JWT Claims </a>,
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
+to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
+<a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
+      </p>
+If <code>kid</code> is also present, it is expected to be useful to distinguish the specific key used.
+      </p>
```

Comment:
As above...
Blend 2 paragraphs? Relocate errant `<p>`? Details about `kid`?
```suggestion
If <code>cnf</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>
or the <a data-cite="RFC7519#section-4.1.1">JWT Claims </a>,
a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
<a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
      </p>
      <p>
If <code>kid</code> is also present, it is expected to be useful to distinguish the specific key used.
      </p>
```

Reply:
Your changes for this and the section above are excellent.

Reply:
Still need to address how `kid` is `to be useful to distinguish the specific key used`, as earlier